README.md

```diff
@@ -1271,6 +1271,9 @@ sudo systemsetup -setrestartfreeze on
 
 #### Chime When Charging
 Play iOS charging sound when MagSafe is connected.
+
+For macOS <= 10.12 (Sierra):
```

Comment:
This needs to read `Up to macOS 10.12 (Sierra)` to match the style of existing entries.